templates/partials/email-form.html

```diff
@@ -0,0 +1,45 @@
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/input/_macro.njk" import onsInput %}
+{% set errors = form.errors['email'] %}
+{% set email_field = form.email_field %}
+
+{% if errors %}
+  {% set error = {
+    "text": errors[0],
+    "id": email_field.id ~ '-error',
+    "attributes": {
+      "data-ga": "error",
+      "data-ga-category": "Error",
+      "data-ga-action": errors[0] | striptags,
+      "data-ga-label": email_field.id
+    }
+  }%}
+{% endif %}
+
+{% set config = {
+  "id": email_field.id,
+  "name": email_field.name,
+  "value": email_field.data or "",
```

Comment:
This value needs to be escaped, as it is in the textfield template. 

```suggestion
  "value": email_field.data | e or "",
```

Otherwise using something like `"><strong>Injected text</strong><p` as a value results in:

![image](https://user-images.githubusercontent.com/1754822/92454949-00082900-f1b9-11ea-8a97-d512c7c143a4.png)

A test should also be added to cover this.